### PR TITLE
nvuApp: changes to connection pool and bug fix

### PR DESCRIPTION
**Pull request by VasilMitovClouway**
Changes to the connection pool string so that now the DB works with cyrilic symbols and a minor bug fix regarding the generation of questionnaires and the displaying of questions registered by users.

### CODE REVIEW

File: src/main/java/http/controllers/ShowRegisterQuestionnaireHandler.java [hunk 1]
```diff
@@ -26,7 +26,7 @@ public ShowRegisterQuestionnaireHandler(QuestionnaireRepository repository) {
   @Override
   public Response handle(Request req) {
     Questionnaire questionnaire = repository.getLastOrNewQuestionnaire();
-      if (questionnaire.noAnswers()) {
+      if (questionnaire.noAnswers()&&questionnaire.noQuestions()) {
```
Comment (mgenov) [inline src/main/java/http/controllers/ShowRegisterQuestionnaireHandler.java]: Seems wrong. We already discussed this on another PR.

Comment (VasilMitovClouway) [inline src/main/java/http/controllers/ShowRegisterQuestionnaireHandler.java]: There are 3 types of Questionnaires:
Type 1 brand new Questionnaire - When the last questionnaire in the DB has answers set up and it has been finished the user will get a new questionnaire here that has no questions and and the correct answers to them whatsoever.
Type 2 questionnaire that is in progress but was left unfinished When last questionnaire in the DB has questions in it but there are no correct answers attached to it ( ie it hasnt been finished )
Type 3 questionnaire that was finished on the last iteration. 
Both type 1 and 2 share a common thing that the correct answers field is not present. The way that i see we can differentiate them is to check if it has no questions at all thus saying that this is a brand new questionnaire.

Comment (mgenov) [inline src/main/java/http/controllers/ShowRegisterQuestionnaireHandler.java]: If so, then: 
```java
If (questionnaire.isEmpty()) {
 
}
```

looks much readable then: if(questionnaire.noAnswers() && questionnaire.noQuestions()) {}

Comment (VasilMitovClouway) [inline src/main/java/http/controllers/ShowRegisterQuestionnaireHandler.java]: I agree. Both conditions have been included in a single method isEmpty()

**Pull request merged**